File: help/pt_pt/exercicios_da_ficha_7/Ficha07_002/Ficha07_002.cs
// Programmer           Ant0ni0 M@ns0 - Ant0ni0 M@ns0

using System.IO;
using System;
public class Ficha07_002 {

    //Main Function
    static void Main() {
        Console.Write("Somatório passo 2\n" );
        int soma = 0 ;
        for( int i = 1; i < 10; i = i + 2 ) {
            Console.Write(soma + " + " + i );
            soma +=i;
            Console.Write(" \t= " + soma + "\n" );
        }
        Console.Write("\nSoma = " + soma );
    }

}
//C#  - Automatic translated by Algorithmi 22.05
//(c) Ant0nio M@nso 	Mon, 2022 Sep 26 16:13

File: help/pt_pt/exercicios_da_ficha_7/Ficha07_002/Ficha07_002.c
// Programmer           Ant0ni0 M@ns0 - Ant0ni0 M@ns0

#include <stdio.h>
#include <string.h>
#include <math.h>
//Main Function
int main() {
    printf("Somatório passo 2\n");
    int soma = 0 ;
    int i = 0 ;
    for( int i = 1; i < 10; i = i + 2 ) {
        printf("%d + %d",soma ,i );
        soma +=i;
        printf(" \t= %d\n",soma );
    }
    printf("\nSoma = %d",soma );
    return 0;
}


//C Language c11  - Automatic translated by Algorithmi 22.05
//(c) Ant0nio M@nso 	Mon, 2022 Sep 26 16:13
